omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -858,7 +858,11 @@
 default['private_chef']['dark_launch']['sql_users'] = true
 default['private_chef']['dark_launch']['add_type_and_bag_to_items'] = true
 default['private_chef']['dark_launch']['reporting'] = true
-default['private_chef']['dark_launch']['actions'] = true
+# It appears that actions rabbitmw was used for oc_actions and analytics.
+# Both those products are deprecated.
```

Comment:
analytics is EOL, not deprecated. This is one of the few places we have specific definitions.

https://docs.chef.io/versions.html#end-of-life-eol